### PR TITLE
{bio}[foss/2021a] sinto v0.7.4  Python 3.9.5

### PR DESCRIPTION
**Pull request by ysagon**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'sinto'
+
+version = '0.7.3.1'
+
+versionsuffix = '-Python-%(pyver)s'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb:7]: '-Python-%(pyver)s' should no longer be included in versionsuffix (when using the default Python for that toolchain)

File: easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Subset reads from a BAM file by cell barcode
+
+    Create a scATAC-seq fragments file from a BAM file
+
+    Add read tags to a BAM file according to cell barcode information
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = [PYPI_SOURCE]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb:22]: 'source_urls' should not be defined when using the default value

File: easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Pysam', '0.16.0.1'),
+    ('SciPy-bundle', '2020.11'),
+]
+
+download_dep_fail = True
+use_pip = True
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb:34]: sanity_pip_check should be enabled

**Review by migueldiascosta — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'sinto'
+
+version = '0.7.4'
+
+homepage = 'https://timoast.github.io/sinto/index.html'
+
```
Comment (bedroge) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb:8]:
```suggestion
name = 'sinto'
version = '0.7.4'

homepage = 'https://timoast.github.io/sinto/index.html'
```

File: easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+description = """Sinto is a toolkit for processing aligned single-cell data. Sinto includes functions to:
+
+    Subset reads from a BAM file by cell barcode
+
+    Create a scATAC-seq fragments file from a BAM file
+
+    Add read tags to a BAM file according to cell barcode information
+"""
```
Comment (bedroge) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb:16]: I think I would either just go for:
```suggestion
description = """Sinto is a toolkit for processing aligned single-cell data."""
```
or I'd add the entire list of functions that they also include on their site?

File: easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    ('Python', '3.9.5'),
+    ('Pysam', '0.16.0.1'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+download_dep_fail = True
+
+use_pip = True
+
+sanity_pip_check = True
```
Comment (bedroge) [inline easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb:33]:
```suggestion
use_pip = True
download_dep_fail = True
sanity_pip_check = True
```

**Review by bedroge — DISMISSED** [review]
Just some very minor style remarks (mostly removing blank lines between parameters that are usually grouped together), other than that it looks fine.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3147.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/b729a7e9f8181fc54803c301a2476c77 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @ysagon!

**Pull request merged**